package.json

```diff
@@ -10,9 +10,14 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "autoprefixer": "^10.4.19",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
```

Comment:
Use ESLint to be more consistant in your code. I had no time to do it, but Prettier is also a must for writing consistant code. It's a bit harder to align these two tools, but in the end it's totally worth it. If you have figured it out and adapted the configuration in one of your projects, use it as a base to start creating new projects from that.

Ideally also combine it with a pre-commit hook like Lint-staged and Husky, so that commited code will automatically be checked and even fixed.

Reply:
Thanks, @iteratus. I had used it once and to be sincere it was tough joining the two but I did, I'll surely refer to the project, normally I add the linters as GH actions and use prettier locally, but this surely saves lots of linter errors in my code.